app/pages/reporter/ciip-application-legal-disclaimer.tsx

```diff
@@ -38,13 +38,13 @@ class CiipApplicationLegalDisclaimer extends Component<Props> {
 
     let ImportMessage = null;
 
-    if (hasImportedReport) {
+    if (!hasImportedReport) {
```

Comment:
Whis is this changed? It doesn't seem right

Reply:
Testing artifact... This is why code reviews are important :)